CONTRIBUTING.md

```diff
@@ -6,35 +6,36 @@ The best way to start is to join our [Forum](https://forum.subspace.network/) or
 [Discord](https://discord.gg/subspace-network), [official website](https://subspace.network/) has a bunch of learning
 materials too.
 
-GitHub is primarily used by developers to work on new features, improvements and fix bugs. If you're not sure whether
-you have an actual bug that needs to be fixed or you just want to ask a question, forum is the best place for that.
+GitHub is primarily used by developers to work on new features and improvements, and to fix bugs. If you're not sure whether
+you have an actual bug that needs to be fixed or you just want to ask a question, the forum is the best place for that.
 
 This document describes expectations and recommendations that should make contributions such as issues and pull requests
 on GitHub productive for everyone involved.
 
 ## Process optimized for maintainers
 
-Autonomys Network is maintained by a small group of passionate individuals with limited time. Number of potential outside
-contributors is much larger than number of maintainers and wider community of users is even bigger than that.
+Autonomys Network is maintained by a small group of passionate individuals with limited time. The number of potential outside
+contributors is much larger than number of maintainers, and the wider community of users is even bigger than that.
 
 As such, the process of contributing to Autonomys Network needs to be optimized in such a way that it doesn't overwhelm
-maintainers with unnecessary notifications, questions that can be answered by wider community and follow practices
-defined in this document below, such that contributions can be processed in the most productive way saving time and
-making it a better experience overall.
+maintainers with unnecessary notifications, or questions that can be answered by wider community. Following the practices
+defined in this document below, such that contributions can be processed in the most productive way, saves time and
+makes for a better experience overall.
 
 ## Bugs, improvements and feature requests
 
 These should be submitted in such a way that they are actionable by developers. Again, if you're unsure or just have a
-question - use forum, GitHub issue can be created later if it is necessary.
+question - use the forum, a GitHub issue can be created later if it is necessary.
 
-Always search for similar topics already created on the forum or here on GitHub before creating a new one, but don't
-hijack threads that looks somewhat similar, but are not necessarily the same or have the same root cause.
+Always search for similar topics already created on the forum or here on GitHub, before creating a new one. But don't
+hijack threads that look somewhat similar, they are not necessarily the same, and might not have the same root cause.
 
 ### Bugs
 
 For bug report to be actionable it should ideally follow the following template:
 
 <details>
+<summary>Click to see the template</summary>
```

Comment:
In rendered markdown the default is "Details" and a small triangle, which isn't clear in the middle of a long document.